### PR TITLE
creates global-reach section of front page as well as all styles for it

### PR DESCRIPTION
**Pull request by jordanjustice**
Hey @jonmspencer -

So I've pushed up the "global reach" section.  A few notes:

* I saved the background image as a `.jpg` at 60% from photoshop. I'm not sure of your team's process here, so I'm happy to change this if needed.
* The `1.5` in the comp has a rounded period and on the web it displays a square period. This might be because of different versions of Roboto but I'm not 100% sure.
* For the secondary mobile sizing, I've resized the background image to only include the middle section.
* I've tried pretty hard to follow your standard here that we covered. One piece I wasn't sure of was your ordering of the css properties, so I did them by type (mixins, position, box model, color, text, misc)

Thanks!

### CODE REVIEW

**Comment by jonmspencer** [comment]
@jordanjustice Thanks for the pr, and notes! 

## Responses to your notes
- The background image in this case is totally cool at a 60% jpg. Looks good!
- Good attention to detail on the period! That's a great catch, but we'll let that slide for the moment and keep that in mind for later. 
- For the mobile sizing, that image solution looks good for now. :) 
- Also, your scss ordering looks completely fine. No worries.

I have a few development notes to talk about, but I'd like to address them over a ScreenHero.

File: wp-content/themes/cloudnine/front-page.php [hunk 1]
```diff
@@ -2,6 +2,18 @@
   get_header();
 ?>
 
+  <section class="global-reach">
+    <h3>Global Reach</h3>
+    <div class="content">
+      <p class="large-number">1.5M+</p>
```
Comment (jordanjustice) [inline wp-content/themes/cloudnine/front-page.php]: @jonmspencer - You mentioned the semantics of this part. Do you guys ever use non-text tags like plain old divs or spans for text on the page? Or do you mainly use h1-6 and p tags?

Comment (jonmspencer) [inline wp-content/themes/cloudnine/front-page.php]: @jordanjustice a `div` or `span` is far preferable to a `p` in this case. For both the `large-number` and the label below it, they act like blocks, so I would opt for a `div`, since it exhibits this behavior by deafult. :)

**Comment by jordanjustice** [comment]
@jonmspencer - I've fixed up the semantics and changed the classes names a little bit. Normally I would use the BEM approach to class names, so in this case the stat elements would have looked something like this:

```html
<div class="stat">
  <div class"stat__number">...</div>
  <div class"stat__subtitle">...</div>
</div>
```

But that doesn't really seem consistent with the project so I refrained.

Let me know what you think of all this. Thanks!

File: wp-content/themes/cloudnine/assets/styles/modules/sections/_global-reach-section.scss [hunk 2]
```diff
@@ -12,36 +12,35 @@ section {
...
         //// Primary Mobile - Tablet & Below
         @media screen and (max-width: $tablet){
           font-size: 6.375em;
         } // End Tablet & Below
       } // End .large-numbergit
 
-      p:last-of-type {
+      .stat-subtitle {
         font-family: $font-light;
         font-style: italic;
```
Comment (jonmspencer) [inline wp-content/themes/cloudnine/assets/styles/modules/sections/_global-reach-section.scss]: @jordanjustice I noticed that there wasn't a var defined for `$font-light-italic`, so I am making one, then we can use it on the `.stat-subtitle` instead of mofifying the regular light webfont.

**Comment by jonmspencer** [comment]
@jordanjustice this is looking good overall! I made a couple tweaks, (mainly centering stats on mobile, which is a last minute design call) but that's about it. Please take a look at the little changes I made, but I'm merging up!
